extensions/standard-processors/processors/TailFile.h

```diff
@@ -71,7 +71,7 @@ enum class Mode {
   SINGLE, MULTIPLE, UNDEFINED
 };
 
-class TailFile : public core::Processor {
+class TailFile : public core::Processor, public core::InputRequirementAnnotation<core::EInputRequirement::INPUT_FORBIDDEN> {
```

Comment:
we could introduce an `annotation` namespace in `core` and provide type aliases for these, like `core::annotation::InputForbidden`, this might make it more compact

Reply:
As we could have more types of annotations (other than input), I slightly modified it to:
core::annotation::input::Forbidden